lib/Raven/Client.php

```diff
@@ -559,9 +559,22 @@ protected function get_http_data()
         if (!empty($_POST)) {
             $result['data'] = $_POST;
         }
+        
+        // if the $_POST super global is empty, try to retrieve a json string
+        // prior to PHP 5.6, a stream opened with php://input could only be read once, so only do this with 5.6 or up
+        if (version_compare(PHP_VERSION, '5.6', '>=') && empty($_POST)) {
+            try {
+                $data = file_get_contents('php://input');
+                $result['data'] = json_decode($data, true);
```

Comment:
No, we need to pull raw text instead of interpret it.
By the way, what is transferred if json will contain a object?  I think it\'s a string
```
[Object object]
```